service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/auth/IotHubServiceSasToken.java

```diff
@@ -20,7 +20,7 @@
  */
 public final class IotHubServiceSasToken
 {
-    private static long DEFAULT_TOKEN_LIFESPAN_SECONDS = 60 * 60; // 1 hour
+    private static long DEFAULT_TOKEN_LIFESPAN_SECONDS = 365*24*60*60; // 1 year
```

Comment:
we went back to 1 year?

Reply:
Was it not 1 year always?

Reply:
For the case where a user provides a connection string rather than a TokenCredential or AzureSasCredential, I reverted back to the old behavior, yes. In this scenario, it would take a lot of code to add proactive renewal + shorter SAS tokens to the old use case. For now, I've decided that it can be treated as an enhancement later, if we need it

Reply:
Essentially, it would more work than I'd like to enhance the existing constructor behavior, so I went back to the old way.